hw_02.09.23/task2/imported.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+new_path = r"D:\PycharmProjects\beetroot\hw_02.09.23\task2"
+sys.path.append(new_path)
+
+print(new_path)
```

Comment:
Please try to import anything from a directory you appended as a `new_path` to `sys.path`. Does it work? Does removing the directory from `sys.path` affect further imports?